shared/general/persistent-storage.md

```diff
@@ -1,10 +1,12 @@
 
-If you want specific data or configurations to persist on the device through the update process, you will need to store them in `/data` . This is a special folder on the device file system which is essentially a [Docker data `VOLUME`][docker-volume-link].
+If you have data or configurations that you want to persist through the update process, you can store them in the `/data` folder. For applications with a single container, this folder is automatically linked to a directory on the host OS, and it is guaranteed to be maintained when new versions of the application or the host OS are applied (for multicontainer applications, you'll want to [configure this link][multicontainer] as appropriate). The `/data` folder is a good place to write system logs and other application data that should be available across updates and services.
 
-This folder is guaranteed to be maintained across updates and thus files contained in it can act as persistent storage.	This is a good place to write system logs, etc.
+Note that the `/data` folder is __not__ mounted when your project is building on our build servers, so you can't access it from your `Dockerfile`. The `/data` volume only exists when the container is running on the deployed devices. 		
```

Comment:
We can add, though, that any data added in /data during the build will be copied into the volume the first time it’s created on the device (which is the default behavior for docker volumes) - for devices on resinOS 2.12 and higher.